comit/src/expiries.rs

```diff
@@ -35,104 +40,140 @@ pub struct Expiries<A, B> {
     alpha_connector: A,
     /// Beta blockchain connector for getting the current time.
     beta_connector: B,
-    /// The alpha ledger relative expiry time.
-    alpha_expiry: Duration,
-    /// The beta ledger relative expiry time.
-    beta_expiry: Duration,
+    /// The alpha ledger expiry offset.
+    alpha_offset: AlphaOffset,
+    /// The beta ledger expiry offset.
+    beta_offset: BetaOffset,
+}
+
+/// Calculate a pair of useful expiries, 'useful' means a swap can be
```

Comment:
Thanks for the comment, I'm not against changing the terminology however using `achievable` would still require a definition. The term 'useful' has been used and defined since the starting writing the spike. Happy to change it everywhere if we can find a term that does not require explaining though.